src/tsickle.ts

```diff
@@ -1596,6 +1634,22 @@ class ExternsWriter extends ClosureRewriter {
     switch (node.kind) {
       case ts.SyntaxKind.SourceFile:
         const sourceFile = node as ts.SourceFile;
+        if (ts.isExternalModule(sourceFile)) {
+          // This file is a d.ts that is a module, so any of its declarations
+          // should be emitted into a namespace.
+          // The actual chosen namespace doesn't matter too much because we're
```

Comment:
What does this do to re-opening types/namespaces? Is that possible at all with external modules?